android/src/main/kotlin/com/signify/hue/flutterreactiveble/ble/ReactiveBleClient.kt

```diff
@@ -70,7 +70,7 @@ open class ReactiveBleClient(private val context: Context) : BleClient {
                 *filters
         )
                 .map { result ->
-                    ScanInfo(result.bleDevice.macAddress, result.bleDevice.name
+                    ScanInfo(result.bleDevice.macAddress, result.scanRecord.deviceName
```

Comment:
I specifically [mentioned](https://github.com/PhilipsHue/flutter_reactive_ble/issues/105#issuecomment-693328205) falling back to the "standard" device name to be on the safe side.

Is there a reason _not_ to do that on Android?

Reply:
whoops forgot to push